### ISSUE TITLE
Error when running `Ogc::Gml::Dictionary.from_xml`

### ISSUE THREAD

**Issue by kwkwan** [lutaml/ogc-gml#11]
The following lines are the steps to reproduce the error:

```
require "ogc/gml"

xml = <<XML
<?xml version=\"1.0\" encoding=\"UTF-8\"?>
<gml:Dictionary xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"
  xmlns:gml=\"http://www.opengis.net/gml/3.2\" xsi:schemaLocation=\"http://www.opengis.net/gml http://schemas.opengis.net/gml/3.1.1/profiles/SimpleDictionary/1.0.0/gmlSimpleDictionaryProfile.xsd\" gml:id=\"cl_c9fa7a39-966f-4ee4-8102-91fb15ad2dd3\">
  <gml:name>Building_class</gml:name>
  <gml:dictionaryEntry>
    <gml:Definition gml:id=\"Building_class_1\">
      <gml:description>普通建物</gml:description>
      <gml:name>3001</gml:name>
    </gml:Definition>
  </gml:dictionaryEntry>
  <gml:dictionaryEntry>
    <gml:Definition gml:id=\"Building_class_2\">
      <gml:description>堅ろう建物</gml:description>
      <gml:name>3002</gml:name>
    </gml:Definition>
  </gml:dictionaryEntry>
  <gml:dictionaryEntry>
    <gml:Definition gml:id=\"Building_class_3\">
      <gml:description>普通無壁舎</gml:description>
      <gml:name>3003</gml:name>
    </gml:Definition>
  </gml:dictionaryEntry>
  <gml:dictionaryEntry>
    <gml:Definition gml:id=\"Building_class_4\">
      <gml:description>堅ろう無壁舎</gml:description>
      <gml:name>3004</gml:name>
    </gml:Definition>
  </gml:dictionaryEntry>
  <gml:dictionaryEntry>
    <gml:Definition gml:id=\"Building_class_5\">
      <gml:description>分類しない建物</gml:description>
      <gml:name>3000</gml:name>
    </gml:Definition>
  </gml:dictionaryEntry>
</gml:Dictionary>
XML

Ogc::Gml::Dictionary.from_xml(xml)
```

The trace outputs:
```
~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:361:in `block in apply_xml_mapping': Attribute 'bounded_by' not found in Ogc::Gml::Definition (RuntimeError)
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:360:in `each'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:360:in `apply_xml_mapping'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:305:in `apply_mappings'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/attribute.rb:201:in `cast'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:394:in `normalize_xml_value'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:371:in `block in apply_xml_mapping'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:360:in `each'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:360:in `apply_xml_mapping'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:305:in `apply_mappings'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/attribute.rb:201:in `cast'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/attribute.rb:198:in `block in cast'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/attribute.rb:197:in `map'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/attribute.rb:197:in `cast'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:394:in `normalize_xml_value'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:371:in `block in apply_xml_mapping'
from ~/.rbenv/versions/3.1.5/lib/ruby/gems/3.1.0/gems/lutaml-model-0.3.12/lib/lutaml/model/serialize.rb:360:in `each'
... 9 levels...

```

**Comment by ronaldtse** [comment]
@kwkwan is this a new bug? Or is the content new? Thanks.

**Comment by kwkwan** [comment]
> @kwkwan is this a new bug? Or is the content new? Thanks.

Yes, this bug appears on and after v1.0.1.

**Comment by HassanAkbar** [comment]
This is fixed in `Lutaml::Model` version `0.3.14`.

**Comment by ronaldtse** [comment]
Confirmed fixed. Thank you @HassanAkbar !
